# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1546885237 18000
#      Mon Jan 07 13:20:37 2019 -0500
# Branch stable
# Node ID 7647b82f921b7d7d42c23f90389966170978792f
# Parent  9f9aa7e92826e2f45e2662b105c2a287c65a369f
don't increment line number twice when parsing continuation markers

* lex.ll (<COMMAND_START>(\.\.\.){ANY_EXCEPT_NL}*{NL}): Don't set line
and column info twice (it is already done by
HANDLE_STRING_CONTINUATION).

diff --git a/libinterp/parse-tree/lex.ll b/libinterp/parse-tree/lex.ll
--- a/libinterp/parse-tree/lex.ll
+++ b/libinterp/parse-tree/lex.ll
@@ -384,19 +384,16 @@ ANY_INCLUDING_NL (.|{NL})
 // If an argument is in construction, it is completed.
 %}
 
 <COMMAND_START>(\.\.\.){ANY_EXCEPT_NL}*{NL} {
     curr_lexer->lexer_debug ("<COMMAND_START>(\\.\\.\\.){ANY_EXCEPT_NL}*{NL}");
 
     COMMAND_ARG_FINISH;
 
-    curr_lexer->m_input_line_number++;
-    curr_lexer->m_current_input_column = 1;
-
     HANDLE_STRING_CONTINUATION;
   }
 
 %{
 // Commands normally end at the end of a line or a semicolon.
 %}
 
 <COMMAND_START>({CCHAR}{ANY_EXCEPT_NL}*)?{NL} {
